testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/mgmt/JMSBridgeManagementTestCase.java

```diff
@@ -0,0 +1,351 @@
+    static final PathAddress ADDRESS_NETTY_ACCEPTOR = ADDRESS_HORNETQ.append(PathElement.pathElement("remote-acceptor", "netty"));
+
+    private List<ServerSetupTask> tasks = new ArrayList<ServerSetupTask>(4);
+    private List<ServerSetupTask> tasksToClean = new ArrayList<ServerSetupTask>(4);
+    
+    public JMSBridgeManagementTestCase(){
+        tasks.add(new DisableHornetQSecruity());
+        tasks.add(new SetupNetty());
+        tasks.add(new SetupConnectionFactory());
+        tasks.add(new SetupQueuesFactory());
+        tasks.add(new SetupJMSBridge());
+    }
+
+    @Before
+    public void setup() throws Exception{
+        for(ServerSetupTask t:this.tasks){
+            this.tasksToClean.add(t);
+            t.setup(this.managementClient, "fake");
+        }
+    }
+
+    @After
+    public void cleanup(){
+        try{
+            for(ServerSetupTask sst:this.tasksToClean){
+                sst.tearDown(this.managementClient, "fake");
+            }
+        }catch(Exception e){
+            
+        }
+        this.tasksToClean.clear();
+    }
+
+    @Test
+    public void testStopStart() throws Exception {
+        //actually even checking state seems enough, but lets try a circle.
+        reload(this.managementClient);
+        final ModelControllerClient client = this.managementClient.getControllerClient();
+        ModelNode operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        ModelNode result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+        
+        operation = Util.createOperation(STOP, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertFalse("Bridge should stop!",result.get(RESULT).asBoolean());
+
+        operation = Util.createOperation(START, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        result = client.execute(operation);
+        operation = Util.createOperation(READ_ATTRIBUTE_OPERATION, PathAddress.pathAddress(ADDRESS_JMS_BRIDGE));
+        operation.get(INCLUDE_RUNTIME).set(true);
+        operation.get(NAME).set(STARTED);
+        result = client.execute(operation);
+        Assert.assertTrue("Bridge should start!",result.get(RESULT).asBoolean());
+    }
+
+    @Override
+    protected ModelControllerClient getModelControllerClient() {
+        return this.managementClient.getControllerClient();
+    }
+    public static void reload(final ManagementClient managementClient) throws Exception {
```

Comment:
yey! yet another reload impl, 
and most likely another problem in our testsuite. 
Using something like this:
```
       final ModelNode operation = new ModelNode();
        operation.get(OP).set("reload");
        operation.get(OP_ADDR).set(address);
        operation.get(BLOCKING).set(true);
```
should be sufficient